modules/candidate_list/jsx/onLoad.js

```diff
@@ -0,0 +1,62 @@
+/* global formatColumn */
+
+import AccessProfilePanel from './AccessProfilePanel';
+import CandidateListFilter from './CandidateListFilter';
+
+class CandidateListPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'filters': { }
+    };
+	this.updateFilter = this.updateFilter.bind(this);
+  }
+
+  updateFilter(filter) {
+    var f = {};
```

Comment:
Other modules use : `this->setState({filter});` But they are also using LorisForm.php to provide the Data.form elements for the FilterForm.

Reply:
I don't understand this comment. There's a `this.setState` immediately below?

Reply:
see: https://github.com/aces/Loris/blob/17.1-dev/modules/media/jsx/mediaIndex.js#L51